src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -178,7 +178,7 @@ export const Runtimes = (signal: AbortSignal) => {
     },
 
     invalidateCookie: () => {
-      return fetchLeo('proxy/invalidateToken', _.merge(authOpts(), { signal }));
+      return fetchLeo('proxy/invalidateToken', _.merge(authOpts(), { signal, credentials: 'include' }));
```

Comment:
Was this th fix that made it work in the UI?

Reply:
yes!

Reply:
Also worth noting that the endpoint wasn't actually ever called in the UI before this PR